### ISSUE TITLE
Error mismatch in test for redis-4.0.11-3.2.x86_64

### ISSUE THREAD

**Issue by Tux** [PerlRedis/perl-redis#135]
```
t/04-pipeline.t ........... 1/?
#   Failed test 'pipeline with embedded error'
#   at t/04-pipeline.t line 42.
# Compared $data->[1][1]
#    got : 'ERR unknown command `OOPS`, with args beginning with: '
# expect : 'ERR unknown command 'OOPS''
t/04-pipeline.t ........... 7/? # Looks like you failed 1 test of 11.
t/04-pipeline.t ........... Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/11 subtests
```
Running `redis-4.0.11-3.2.x86_64`
``` sh
$ grep -w OOPS t/04-pipeline.t
  ([set => [clunk => 'eth'], 'OK'], [oops => [], undef, q[ERR unknown command `OOPS`, with args beginning with: ]], [get => ['clunk'], 'eth'],);
$ make test
:
All tests successful.
Files=18, Tests=281, 70 wallclock secs ( 0.11 usr  0.03 sys +  6.04 cusr  1.32 csys =  7.50 CPU)
Result: PASS
```

**Comment by shogo82148** [comment]
hi, I'm a maintainer of [Redis::Fast](https://metacpan.org/pod/Redis::Fast).
Redis::Fast had same issue https://github.com/shogo82148/Redis-Fast/issues/80, and it is fixed by https://github.com/shogo82148/Redis-Fast/pull/82 .

**Comment by shogo82148** [comment]
It seems that this issue was fixed in master branch https://github.com/PerlRedis/perl-redis/pull/130, but not released.

@dams could you make a new release including this fix?

**Comment by dams** [comment]
released in 1.992
